packages/components/src/mobile/bottom-sheet/navigation-container.native.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { BottomSheetContext, BottomSheetProvider } from '@wordpress/components';
+import { useState, useContext } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import { performLayoutAnimation } from '../layout-animation';
+
+function BottomSheetNavigationContainer( { children, animate } ) {
+	const [ height, setMaxHeight ] = useState( 1 );
```

Comment:
Not suggesting a change here, just wondering why we start with 1, and also why are the getter and setter named differently?